gradle.properties

```
@@ -2,14 +2,14 @@
 org.gradle.jvmargs=-Xmx1G
 
 # Fabric Properties
-minecraft_version=20w09a
-yarn_mappings=20w09a+build.2
-loader_version=0.7.8+build.184
+minecraft_version=20w12a
+yarn_mappings=20w12a+build.5
+loader_version=0.7.8+build.188
 
 #Fabric api
-fabric_version=0.4.33+build.301-1.16
+fabric_version=0.5.5+build.311-1.16
 
 # Mod Properties
-mod_version = 1.3
+mod_version = 1.4
```

```suggestion
mod_version = 1.3
```
No need to increment the mod version if nothing changes.